services/GraphQL/mutations/users.ts

```diff
@@ -0,0 +1,33 @@
+import { gql } from '@apollo/client'
+
+// create user
+export const CREATE_USER = gql`
+  mutation ($username: String!, $password: String!, $email: String!, $type: String!) {
+    insert_users_one(
+      object: { username: $username, password: $password, email: $email, type: $type, is_active: true }
+    ) {
+      affected_rows
+    }
+  }
+`
+
+// update user
+export const UPDATE_USER = gql`
+  mutation ($id: String!, $username: String, $password: String, $email: String, $type: String) {
+    update_users_by_pk(
+      pk_columns: { id: $id }
+      _set: { username: $username, password: $password, email: $email, type: $type }
+    ) {
+      affected_rows
+    }
+  }
+`
+
+// hide user
+export const HIDE_USER = gql`
+  mutation ($id: String!) {
+    update_users_by_pk(pk_columns: { id: $id }, _set: { is_active: false }) {
+      affected_rows
+    }
+  }
```

Comment:
nit: `Hide` sounds to me as CSS `display: none`, maybe a more specified name like: SOFT_DELETE_USER ?

Reply:
Hmm what about "deactivate"?

Or @e-azocar what do you think?

Reply:
I think it's name could be `DISABLE_USER`

Reply:
Seems like everyone have different opinion hehe

Maybe we have to confirm with Ana for example, like what word is understand fr this purpose, what do you think guys?

Reply:
@anaceciliamolinares what you think is the correct word to make a user not being taken into account, but the data is still available if we need to make it appear again?

Reply:
`deactivate` or `disable_user` sounds good to me

Reply:
@Israel-Laguan Let's merge this pull requests an create a new Issue for this variable name. I've noticed too a "hide_event" and "hide_provider"